core/main.go

```diff
@@ -177,30 +177,34 @@ func (a *AutoSpotting) getRegions() ([]string, error) {
 
 // convertRawEventToCloudwatchEvent parses a raw event into a CloudWatchEvent or
 // returns an error in case of failure
-func convertRawEventToCloudwatchEvent(event *json.RawMessage) (*events.CloudWatchEvent, error) {
-	var snsEvent events.SNSEvent
+func (a *AutoSpotting) convertRawEventToCloudwatchEvent(event *json.RawMessage) (*events.CloudWatchEvent, error) {
+	var sqsEvent events.SQSEvent
 	var cloudwatchEvent events.CloudWatchEvent
 
 	log.Println("Received event: \n", string(*event))
 	parseEvent := *event
 
-	// Try to parse event as an Sns Message
-	if err := json.Unmarshal(parseEvent, &snsEvent); err != nil {
+	// Try to parse event as an Sqs Message
+	if err := json.Unmarshal(parseEvent, &sqsEvent); err != nil {
 		log.Println(err.Error())
 		return nil, err
 	}
 
-	// If the event comes from Sns - extract the Cloudwatch event embedded in it
-	if snsEvent.Records != nil {
-		snsRecord := snsEvent.Records[0]
-		parseEvent = []byte(snsRecord.SNS.Message)
+	// If the event comes from Sqs - extract the Cloudwatch event embedded in it
+	if sqsEvent.Records != nil {
+		sqsRecord := sqsEvent.Records[0]
+		parseEvent = []byte(sqsRecord.Body)
+		a.config.sqsReceiptHandle = sqsRecord.ReceiptHandle
```

Comment:
why do we need to persist this `ReceiptHandle` into the configuration?

Reply:
we need it to delete the message